src/vm/gcenv.ee.cpp

```diff
@@ -338,22 +344,39 @@ void GCToEEInterface::GcEnumAllocContexts(enum_alloc_context_func* fn, void* par
     }
 }
 
-bool GCToEEInterface::IsPreemptiveGCDisabled(Thread * pThread)
+bool GCToEEInterface::IsPreemptiveGCDisabled()
 {
     WRAPPER_NO_CONTRACT;
-    return !!pThread->PreemptiveGCDisabled();
+
+    Thread* pThread = ::GetThread();
+    if (pThread)
```

Comment:
Isn't pThread != nullptr better?

Reply:
Depends on your style. In my mind, implicit bool conversions are more standard c++.